src/ResourceManager/Network/Commands.Network/help/New-AzureRmVpnGateway.md

```diff
@@ -0,0 +1,279 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/new-azurermvpngateway
+schema: 2.0.0
+---
+
+# New-AzureRmVpnGateway
+
+## SYNOPSIS
+Creates a Scalable VPN Gateway.
+
+## SYNTAX
+
+### ByVirtualHubName (Default)
+```
+New-AzureRmVpnGateway -Name <String> -ResourceGroupName <String> -VpnGatewayScaleUnit <UInt32>
+ -VirtualHubName <String>
+ [-VpnConnection <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSVpnConnection]>]
```

Comment:
array not list.  apply to all files.

Reply:
Already fixed in code - regenerated help.